x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.tsx

```diff
@@ -206,12 +207,13 @@ const makeMapStateToProps = () => {
     headers: ColumnHeaderOptions[],
     browserFields: BrowserFields
   ) => ColumnHeaderOptions[] = memoizeOne(getColumnHeaders);
-
+  const getEvents = timelineSelectors.getEventsByIdSelector();
```

Comment:
what is the difference between using `getEventsByIdSelector` vs `getTimelineByIdSelector` ?

Reply:
seems to be the same ^ ^"